http_client_wrapper.go

```diff
@@ -89,7 +113,7 @@ func (h httpClientWrapper) ExecuteRequest(path, method string, body []byte) (*ht
 			return errResp
 		}
 		return err
-	}, retry.Attempts(5), retry.Delay(500*time.Millisecond), retry.MaxDelay(9*time.Second))
+	}, retry.Attempts(uint(h.retry)), retry.Delay(500*time.Millisecond), retry.MaxDelay(9*time.Second))
```

Comment:
thoughts on making h.retry a uint type so can avoid conversion, and also more accurate for user (retry cannot be negative)